Compile with HDF5 (recommended) and FFT (recommended for use with OpenMP)
author: Claire Kopenhafer (Michigan State University)
--- arch/makefile.include.linux_nv_omp.orig	2025-10-20 17:01:31.380201392 -0400
+++ arch/makefile.include.linux_nv_omp	2025-10-20 17:00:47.300409402 -0400
@@ -69,12 +69,17 @@
 # For the parser library
 CXX_PARS   = nvc++ --no_warnings
 
-### For the fft library
-##CXX_FFTLIB = nvc++ -mp -std=c++11 -DFFTLIB_THREADSAFE
-##INCS_FFTLIB= -I./include -I$(FFTW)/include
-##LIBS       += fftlib
-##LLIBS      += -ldl
+# For the fft library
+CXX_FFTLIB = nvc++ -mp -std=c++11 -DFFTLIB_THREADSAFE
+INCS_FFTLIB= -I./include -I$(FFTW)/include
+LIBS       += fftlib
+LLIBS      += -ldl
 
 # Normally no need to change this
 SRCDIR     = ../../src
 BINDIR     = ../../bin
+
+CPP_OPTIONS+= -DVASP_HDF5
+HDF5_ROOT  ?= $(EBROOTHDF5)
+LLIBS      += -L$(HDF5_ROOT)/lib -lhdf5_fortran
+INCS       += -I$(HDF5_ROOT)/include
